src/framework/mpas_forcing.F

```diff
@@ -13,7 +13,7 @@
 module mpas_forcing
 
 #define COMMA ,
-#define FORCING_DEBUG_WRITE(M) !call mpas_log_write( M )
+#define FORCING_DEBUG_WRITE(M) call mpas_log_write( M )
```

Comment:
I don't know whether we have guidelines for this sort of thing, but it may be worth leaving this line unchanged.

Reply:
Good point Michael. I went ahead and changed it back. Thanks for catching that!